slash_commands/index-2tc.js

```diff
@@ -225,21 +223,20 @@ async function execute(interaction) {
         new Parsed()
     )
 
-    await interaction.deferReply({ ephermal: true })
```

Comment:
This was a typo I didn't notice lol. I learned that you can't update an ephermal reply to be non-ephemeral and vice versa.